Sources/Internal/WebSocketConnection.swift

```diff
@@ -35,7 +35,7 @@ class WebSocketConnection {
         self.onDisconnect = onDisconnect
         self.onTextReceive = onTextReceive
         serialCallbackQueue = DispatchQueue(label: "org.walletconnect.swift.connection-\(url.bridgeURL)-\(url.topic)")
-        socket = WebSocket(url: url.bridgeURL)
+        socket = WebSocket(request: URLRequest(url: url.bridgeURL))
```

Comment:
This change and in other places in this file is because Starscream API changed in next version. Everything looks good to me.
I think `private var isConnected = false` can be removed in favour of `isOpen`